static/app/views/releases/detail/overview/issues.tsx

```diff
@@ -193,59 +334,88 @@ class Issues extends Component<Props, State> {
   };
 
   render() {
-    const {issuesType, pageLinks, onCursor} = this.state;
+    const {issuesType, count, pageLinks, onCursor} = this.state;
     const {organization, queryFilterDescription, withChart} = this.props;
     const {path, queryParams} = this.getIssuesEndpoint();
+    const hasReleaseComparison = organization.features.includes('release-comparison');
     const issuesTypes = [
-      {value: IssuesType.NEW, label: t('New Issues')},
-      {value: IssuesType.RESOLVED, label: t('Resolved Issues')},
-      {value: IssuesType.UNHANDLED, label: t('Unhandled Issues')},
-      {value: IssuesType.ALL, label: t('All Issues')},
+      {value: IssuesType.NEW, label: t('New Issues'), issueCount: count.new},
+      {
+        value: IssuesType.RESOLVED,
+        label: t('Resolved Issues'),
+        issueCount: count.resolved,
+      },
+      {
+        value: IssuesType.UNHANDLED,
+        label: t('Unhandled Issues'),
+        issueCount: count.unhandled,
+      },
+      {value: IssuesType.ALL, label: t('All Issues'), issueCount: count.all},
     ];
 
     return (
       <Fragment>
         <ControlsWrapper>
-          <DropdownControl
-            button={({isOpen, getActorProps}) => (
-              <StyledDropdownButton
-                {...getActorProps()}
-                isOpen={isOpen}
-                prefix={t('Filter')}
-                size="small"
-              >
-                {issuesTypes.find(i => i.value === issuesType)?.label}
-              </StyledDropdownButton>
-            )}
-          >
-            {issuesTypes.map(({value, label}) => (
-              <StyledDropdownItem
-                key={value}
-                onSelect={this.handleIssuesTypeSelection}
-                data-test-id={`filter-${value}`}
-                eventKey={value}
-                isActive={value === issuesType}
-              >
-                {label}
-              </StyledDropdownItem>
-            ))}
-          </DropdownControl>
+          {hasReleaseComparison ? (
+            <StyledButtonBar active={issuesType} merged>
+              {issuesTypes.map(({value, label, issueCount}) => (
+                <Button
+                  key={value}
+                  barId={value}
+                  size="small"
+                  onClick={() => this.handleIssuesTypeSelection(value)}
+                >
+                  {label}
+                  <QueryCount count={issueCount} max={99} hideParens hideIfEmpty />
```

Comment:
We still want number `0` to show up.
```suggestion
                  <QueryCount count={issueCount} max={99} hideParens hideIfEmpty={false} />
```